README.adoc

```diff
@@ -24,8 +24,7 @@ This architecture brings the following advantages:
 * The JVM for running the actual builds does not need to get started anew for each build.
 * The classloaders holding classes of Maven plugins are cached over multiple builds. The plugin jars are thus read
   and parsed just once. SNAPSHOT versions of Maven plugins are not cached.
-* The native code produced by the Just-In-Time (JIT) compiler inside the JVM is kept too. Compared to stock Maven,
-  less time is spent by the JIT compilation. During the repeated builds the JIT-optimized code is available
+* The native code produced by the Just-In-Time (JIT) compiler inside the JVM is kept too. JIT compilation is faster than stock Maven. During the repeated builds the JIT-optimized code is available
```

Comment:
> JIT compilation is faster than stock Maven.

This changes the meaning in a wrong way. The changed sentence does not state anything useful.